### PR TITLE
Font 2024 queries

### PR DESCRIPTION
**Pull request by konfirmed**
## To be added 

- [ ] Font Usage
- [ ] ...

### CODE REVIEW

**Comment by IvanUkhov** [comment]
Oh, I have just stumbled upon this PR. I have I think this is obsolete due to https://github.com/HTTPArchive/almanac.httparchive.org/pull/3696 and can be closed. The queries from 2022 have been updated to the new data format and reorganized according to the chapter's structure.

File: sql/2024/fonts/color_font_usage.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    date = '2022-06-01' AND
+    type = 'font' AND
+    REGEXP_CONTAINS(JSON_EXTRACT(payload,
+        '$._font_details.color.formats'), '(?i)(sbix|CBDT|SVG|COLRv0|COLRv1)'))
+JOIN (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    COUNT(0) AS total_page
+  FROM
+    `httparchive.pages.2022_06_01_*`
```
Comment (mgifford) [inline sql/2024/fonts/color_font_usage.sql:22]: This should be using `httparchive.all.pages` and the date is still for 2022.

**Review by mgifford — CHANGES_REQUESTED** [review]
Just noting the obvious.

**Comment by mgifford** [comment]
Sorry @IvanUkhov are the fonts being tracked elsewhere? The SQL definitely needs to be rewritten for the new format.

**Comment by IvanUkhov** [comment]
@mgifford, please consider the other PR to be the primary one.

**Pull request closed without merging**